src/module/components/ngx-auth-firebaseui-user/user.component.ts

```diff
@@ -45,23 +45,23 @@ export class UserComponent {
   updatePhoneNumberFormControl: FormControl;
   updatePasswordFormControl: FormControl;
 
-  constructor(@Inject(NgxAuthFirebaseUIConfigToken)
-              public config: NgxAuthFirebaseUIConfig,
-              public auth: AngularFireAuth,
-              public authProcess: AuthProcessService,
-              private _fireStoreService: FirestoreSyncService,
-              private snackBar: MatSnackBar) {
-  }
+  constructor(
+    public auth: AngularFireAuth,
+    public authProcess: AuthProcessService,
+    private _fireStoreService: FirestoreSyncService,
+    private snackBar: MatSnackBar,
+    @Inject(forwardRef(() => NgxAuthFirebaseUIConfigToken)) public config: NgxAuthFirebaseUIConfig
+  ) { }
 
   protected initUpdateFormGroup() {
     const currentUser: User = this.auth.auth.currentUser;
     this.updateFormGroup = new FormGroup({
       name: this.updateNameFormControl = new FormControl(
-        {value: currentUser.displayName, disabled: this.editMode},
+        { value: currentUser.displayName, disabled: this.editMode },
         [
           Validators.required,
-          Validators.minLength(2),
-          Validators.maxLength(25),
+          Validators.minLength(this.config.nameMinLength),
```

Comment:
Did you test that feature ?

I think the config could be here unsync ...

Reply:
I'm not following, could you elaborate please? Thx.